src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/IModelBinder.cs

```diff
@@ -13,7 +13,7 @@ public interface IModelBinder
         /// <summary>
         /// Attempts to bind a model.
         /// </summary>
-        /// <param name="bindingContext">The <see cref="ModelBindingContext"/>.</param>
+        /// <param name="bindingContext">The <see cref="DefaultModelBindingContext"/>.</param>
```

Comment:
S/b `ModelBindingContext`. See a fair few other incorrect mentions of `DefaultModelBindingContext` in doc comments.